android/settings.gradle

```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'rainbow'
+include ':react-native-action-sheet'
+project(':react-native-action-sheet').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-action-sheet/android')
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
```

Comment:
It looks like you manually linked the library.  After RN 0.60+ we shouldn't do that anymore. See: https://medium.com/@srbkrishnan/react-native-auto-linking-on-android-65a850bb9ed9

Reply:
Sorry, unlinked